src/main/java/uk/ac/ebi/embl/api/validation/helper/MasterSourceFeatureUtils.java

```diff
@@ -17,13 +19,26 @@
 	private final  Pattern isolation_sourcePattern = Pattern.compile(isolation_source_regex);
 	private boolean addUniqueName=true;
 	private final Map<String,String> qualifierSynonyms = new HashMap<>();
+	private final Set<String> covid19RequiredQuals = new HashSet<>();
 
 	public MasterSourceFeatureUtils() {
 		qualifierSynonyms.put("metagenomic source","metagenome_source");
+		qualifierSynonyms.put("host scientific name","host");
+		qualifierSynonyms.put("gisaid accession is","note");
+		qualifierSynonyms.put("geographic location (country and/or sea)","country");
+		qualifierSynonyms.put("geographic location (region and locality)","country");
+		covid19RequiredQuals.add(Qualifier.COLLECTION_DATE_QUALIFIER_NAME);
+		covid19RequiredQuals.add(Qualifier.COUNTRY_QUALIFIER_NAME);
+		covid19RequiredQuals.add(Qualifier.LAT_LON_QUALIFIER_NAME);
+		covid19RequiredQuals.add(Qualifier.HOST_QUALIFIER_NAME);
+		covid19RequiredQuals.add(Qualifier.NOTE_QUALIFIER_NAME);
```

Comment:
I think NOTE_QUALIFIER_VALUE is not correct. Jeena says:

```
GISAID Accession ID
/note="EPI_ISL_415509"
this is a user-defined attribute. Can this be mapped to the flat file? If so will need to be case-insenstive.
```

i.e. the value does not come from a sample field called 'note'. Might be a good idea to ask for one example from Jeena?